src/ImageInTerminal.jl

```diff
@@ -39,10 +39,9 @@ Call `ImageInTerminal.use_256()` to restore default behaviour.
 use_24bit() = (colormode[1] = TermColor24bit())
 
 # colorant arrays
-function Base.show(
-        io::IO, ::MIME"text/plain",
+function Base.print_matrix(
+        io::IO,
         img::AbstractVecOrMat{<:Colorant})
```

Comment:
`AbstractVecOrMat{<:Colorant})` is `Union{AbstractArray{T, 1}, AbstractArray{T, 2}}`

So `Array{T, 3}` doesn't go into this method directly; I guess `SparseArrays` has defined a `print_matrix(AbstractArray)` which splits `Array{T, 3}` into several `Array{T, 2}`.

Reply:
Yes, `print_matrix` is called both for showing matrices, and by `show_nd` which decides what slices of dims>2 arrays to display, and prints `[:, :, 2, 3]` etc.

Reply:
I'm still skeptical to this change. It doesn't feel right for me that `imshow` for 3d images depends on the implementation of `SparseArrays`. Hmmm, It looks like some kind of type piracy to me.

This comment is likely to have a strong personal opinion. So it's better to have others review it. @Evizero

Reply:
No it's not SparseArrays, this is just reaching into Base's tower of functions used for printing matrices. 

(I think the SparseArrays thing was just an unrelated error, an import which wasn't required on <=1.1 but seems to be on >= 1.3, including master on travis.)

Reply:
That makes sense, thanks for the explanation.